lib/ts/recipe/emailpassword/index.ts

```diff
@@ -153,7 +159,7 @@ export default class Wrapper {
         email: string,
         password: string,
         userContext?: Record<string, any>
-    ): Promise<{ status: "OK" | "WRONG_CREDENTIALS_ERROR" }> {
+    ): Promise<{ status: "OK" | "WRONG_CREDENTIALS_ERROR" | "USER_BANNED_ERROR" }> {
```

Comment:
probably also want to output the user object alongside this. Cause this way, devs can know which user id this belongs too